### PR TITLE
Pull Request for Creating a database based on a diagram

### PR DESCRIPTION
**Pull request by chuaindia**
In this branch the following activities are incorporated :

- A database diagram for a clinic is given. Taking this as a reference the tables with their attributes are made.
- 5 tables are created following the diagram.
- A file named schema_based_on_diagram.sql where you implement the database from the diagram is created.
- A join table from many-to-many relationships between 2 tables is created.
- Foreign key indexing is added.

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,48 @@
+CREATE DATABASE clinic;
+
```
Comment (ShadyShawkat) [inline schema_based_on_diagram.sql:2]: - [x] Please make sure to add indexes for each of the foreign keys (6 in total), as per [this](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/database_from_diagram.md#:~:text=Remember%20to%20add%20the%20FK%20indexes.) requirement.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi Team, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- All linters are passing ✔️ 
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by chuaindia** [comment]
Ok I am making the changes required.

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (chuaindia) [reply schema_based_on_diagram.sql:2]: Indexing added.

**Review by elmar8287 — APPROVED** [review]
Hi.

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 

### Highlights

- The suggestions from the reviewer were implemented 👍 

### Optional suggestions

- N.A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
### Please, remember to tag me (@elmar8287) in your question so I can receive the notification 👍 
giphy

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**